airflow/jobs/triggerer_job_runner.py

```diff
@@ -751,3 +756,31 @@ def get_trigger_by_classpath(self, classpath: str) -> type[BaseTrigger]:
         if classpath not in self.trigger_cache:
             self.trigger_cache[classpath] = import_string(classpath)
         return self.trigger_cache[classpath]
+
+    @staticmethod
+    def add_trigger_cancel_reasons(
```

Comment:
It seems like this is only intended to be used as an internal method. Add a leading underscore to the name (`_add_trigger_cancel_reasons`) to indicate this.

Reply:
Thanks, renamed to _add_trigger_cancel_reasons already